function-controller/Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: build clean dockerize gen-mocks test kubectl-apply
 OUTPUT = function-controller
 GO_SOURCES = $(shell find pkg cmd -type f -name '*.go')
-TAG = 0.0.5-snapshot
+TAG ?= $(shell cat ../VERSION)
```

Comment:
Out of curiousity, why not `$(cat ../VERSION)`?

Reply:
I thought I tried that and it didn't work, but my make skills are weak.

Reply:
And my noticing-it's-not-bash skills are weaker.